### ISSUE TITLE
Sir, Is this repo open for contributions? 

### ISSUE THREAD

**Issue by thesmallstar** [harchani-ritik/scab_flutter#1]

**Comment by harchani-ritik** [comment]
Yeah sir @thesmallstar , you can surely contribute on this repo if you wish to.

**Comment by harchani-ritik** [comment]
It would be helpful if you can look for in-app payments for splitting the total bill either using network gateway or without it using other UPI apps like Gpay.

**Comment by thesmallstar** [comment]
Thank you for the response, I will be looking looking into it.
